politeiawww/cmd/politeiawww_dataload/main.go

```diff
@@ -555,7 +555,13 @@ func login(email, password string) error {
 }
 
 func logout() error {
```

Comment:
logout() should use executeCliCommand(), similar to login(), so that you're not redefining all of those arguments.